node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -147,6 +170,90 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
   def isDisconnected(idx: Int): Future[Boolean] =
     peerMsgSenders(idx).isDisconnected()
 
+  private def createInDbIfBlockFilterPeer(peer: Peer): Future[Unit] = {
```

Comment:
So this seems like it should belong in the `NeutrinoNode` class rather than the `Node` superclass. 

Ideally, we would like to keep generic bitcoin p2p stuff in the `Node` trait, while keeping neutrino specific stuff in the `NeutrinoNode` trait.